encrypted-media-respec.html

```diff
@@ -2024,9 +2024,21 @@
 
     <section>
       <h2><a>HTMLMediaElement</a> Extensions</h2>
+      
+      <!-- Put all the monkey-patching here -->
+      
       <p>This section specifies additions to and modifications of the <a def-id="htmlmediaelement"></a> [[!HTML5]] when the Encrypted Media Extensions are supported.</p>
+      
+      <p>This specification adds two internal values to the <a def-id="htmlmediaelement"></a>: <var>waiting for key</var> which shall have a boolean value and <var>encrypted block queue</var> which shall be a queue of encrypted blocks awaiting decryption.</p>
```

Comment:
On reviewing the text, I don't see a need to update the state in setMediaKeys. That method runs the "Attempt to resume playback if necessary" method, so if we are blocked waiting for a key at this time then this will unblock us (if the right key is there).